core/controllers/suggestion_test.py

```diff
@@ -1090,9 +1087,6 @@ def setUp(self):
 
         # Login and create exploration and suggestions.
         self.login(self.EDITOR_EMAIL)
-        self.save_new_linear_exp_with_state_names_and_interactions(
-            self.EXP_ID, self.editor_id, ['State 1', 'State 2', 'State 3'],
```

Comment:
Why this changes?

Reply:
While fixing issues with this test during development, I realized that this line doesn't actually do anything since the exploration is not published (an exploration with the same ID is published above this line).